src/sass/partials/_sidebar.scss

```diff
@@ -0,0 +1,20 @@
+aside.pf-c-page__sidebar {
+    padding-top: 0px;
+
+    .ins-c-navigation__header {
+        max-height: 4.4rem;
+        overflow: hidden;
+
+        .pf-l-split__item {
+            padding: 20px;
+            border-bottom: var(--pf-global--BorderWidth--sm) solid #cccccc;
+            &:first-child {
+                cursor: pointer;
```

Comment:
We shouldn't need this when the home button is converted to an `a` or a `button`